SDK v2 migration guide.md

```diff
@@ -96,7 +111,7 @@ but users are still encouraged to migrate to version 2 when they have the chance
 
 | Version 1 API | Equivalent version 2 API |
 |:---|:---|
-| `RegistryManager` | `IotHubServiceClient`, subclients `Devices`, `Twins`, `Configurations`, etc. |
+| `RegistryManager` | `IotHubServiceClient`, `subclients`, `Devices`, `Twins`, `Configurations`, etc. |
```

Comment:
```suggestion
| `RegistryManager` | `IotHubServiceClient`, subclients: `Devices`, `Twins`, `Configurations`, etc. |
```

Reply:
subclients isn't an actual type (meaning we don't need the code (``) wrapper around it.